### PR TITLE
Doc updates per conversation btw. Drew/Scott/Matt

### PR DESCRIPTION
**Pull request by matt-winkler**
https://fishtownanalytics.slack.com/archives/G01H940QRQ9/p1607988023003600

Also a question on how to references images in the markdown on our github.

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [ ] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md [hunk 1]
```diff
@@ -78,11 +79,23 @@ the Redirect URI in the steps above, then skip ahead to step 8.
...
+
+Once you've registered the application, the next step is to assign users to it. Add the users you want to be viewable to dbt with the following steps:
+
+8. From the **Default Directory** click **Enterprise Applications**
+9. Click the name of the application you created earlier
+10. Click **Assign Users and Groups**
+11. Click **Add User/Group**
+12. Assign additional users and groups as-needed
+
+# Where do I go to upload the image as referenced in the path below? Not seeing this from navigating this git repo or searching for /img/ as a repository at the top level of the dbt org.
```
Comment (drewbanin) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md]: Images are served from the [website/static/](https://github.com/fishtown-analytics/docs.getdbt.com/tree/current/website/static) directory. So, the image above points to `/img/docs/dbt-cloud/dbt-cloud-enterprise/azure/azure-redirect-uri.png` which is [this file]((https://github.com/fishtown-analytics/docs.getdbt.com/blob/current/website/static/img/docs/dbt-cloud/dbt-cloud-enterprise/azure/azure-redirect-uri.png)

**Review by drewbanin — COMMENTED** [review]
Thanks for making this PR! Let me know if you'd like a hand with regenerating images for these sections.... it's kind of a chore, but I think they're really helpful!

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md (hunk 1 above)
Comment (matt-winkler) [reply website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md]: @drewbanin FYI fixed the image links

Comment (drewbanin) [reply website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md]: @matt-winkler are those image changes in this PR? https://github.com/fishtown-analytics/docs.getdbt.com/pull/500

Would be good to merge these two branches together IMO, but LGTM once the images are being rendered in dev/staging :)

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md [hunk 2]
```diff
@@ -78,11 +79,22 @@ the Redirect URI in the steps above, then skip ahead to step 8.
 
 <Lightbox collapsed="true" src="/img/docs/dbt-cloud/dbt-cloud-enterprise/azure/azure-redirect-uri.png" title="Configuring a Redirect URI"/>
 
+### Adding Users to an Enterprise Application
+
+Once you've registered the application, the next step is to assign users to it. Add the users you want to be viewable to dbt with the following steps:
```
Comment (amychen1776) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md:88]: Could you link to our enterprise permissions? They need to create the dbt Cloud groups as well to map to those Azure AD groups.

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md [hunk 3]
```diff
@@ -157,3 +169,11 @@ Users in your Azure AD account will now be able to log into the application
 by navigating to the URL:
 
 `https://cloud.getdbt.com/enterprise-login/<login-slug>`
+
+### Troubleshooting Tips
+
+Ensure that the domain name under which user accounts exist in Azure matches the domain supplied in the SSO configuration on the dbt side.
+
+<Lightbox collapsed="true" src="/img/docs/dbt-cloud/dbt-cloud-enterprise/azure/azure-get-domain.png" title="Obtaining the user domain from Azure" />
```
Comment (amychen1776) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md:184]: Did you want to include the Troubleshooting Tips in the same box as logging in?
image

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md [hunk 4]
```diff
@@ -157,3 +169,11 @@ Users in your Azure AD account will now be able to log into the application
 by navigating to the URL:
 
 `https://cloud.getdbt.com/enterprise-login/<login-slug>`
```
Comment (amychen1776) [inline website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md:175]: If they're connecting to a single tenant or On-Prem instance, this URL would change the domain away from `cloud.getdbt.com`. Might be useful to flag this.

Comment (matt-winkler) [reply website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md:175]: @amychen1776 Added a new commit with this called out.

Comment (amychen1776) [reply website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md:175]: Thank you!

File: website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md (hunk 2 above)
Comment (matt-winkler) [reply website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory.md:88]: @amychen1776 added a new header and paragraph for this immediately before this section.

**Comment by matt-winkler** [comment]
Changes are now contained in https://github.com/fishtown-analytics/docs.getdbt.com/pull/502

**Pull request closed without merging**